src/LaunchDarkly.Client/LaunchDarkly.Client.csproj

```diff
@@ -37,6 +37,18 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
+    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
+      <HintPath>\\Mac\Home\Documents\packages\Microsoft.Extensions.DependencyInjection.Abstractions.1.0.0\lib\netstandard1.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll</HintPath>
```

Comment:
When I added these dependencies Visual Studio created this hint path, but we want them to be the relative paths within the repo that NuGet creates... I changed it and the AppVeyor build now passes